NullPointerException in NumberListController

How to reproduce:

1. Go to http://localhost:8080/webapp/content and click "View List" for Numbers

   ![Image](https://github.com/user-attachments/assets/7e84e099-e312-4f63-b3a1-8ac67f85fc29)

2. The page (http://localhost:8080/webapp/content/number/list) crashes

   ![Image](https://github.com/user-attachments/assets/5eb5517d-bca8-49b4-b7e8-8a28e64b53d5)

   ```
   Caused by: java.lang.NullPointerException: Cannot invoke "ai.elimu.dao.NumberDao.readAllOrdered()" because 
   "this.numberDao" is null
       at ai.elimu.web.content.number.NumberListController.handleRequest (NumberListController.java:25)
   ```

--- a/src/main/java/ai/elimu/web/content/number/NumberListController.java
+++ b/src/main/java/ai/elimu/web/content/number/NumberListController.java
@@ -16,7 +16,7 @@ import org.springframework.web.bind.annotation.RequestMapping;
 @Slf4j
 public class NumberListController {
     
-    private NumberDao numberDao;
+  private final NumberDao numberDao;
 
   @GetMapping
   public String handleRequest(Model model) {

